core/src/main/java/de/qabel/core/repository/sqlite/SqliteDropStateRepository.kt

```
@@ -0,0 +1,32 @@
+package de.qabel.core.repository.sqlite
+
+import de.qabel.core.repository.DropStateRepository
+import de.qabel.core.repository.EntityManager
+import de.qabel.core.repository.entities.DropState
+import de.qabel.core.repository.exception.EntityNotFoundException
+import de.qabel.core.repository.framework.BaseRepositoryImpl
+import de.qabel.core.repository.sqlite.schemas.DropStateDB
+
+class SqliteDropStateRepository(database: ClientDatabase,
+                                entityManager: EntityManager) :
+    BaseRepositoryImpl<DropState>(DropStateDB, database, entityManager), DropStateRepository {
+
+    override fun getDropState(dropId: String) = findByDropId(dropId, true)!!.eTag
```

forcoNotNull throws an exception instead of returning null. The forceNotNull flag should be removed.

An extension method can do that job just fine.